### PR TITLE
opencl-clhpp-headers: v2.0.19

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/opencl-headers.rb [hunk 1]
```diff
@@ -21,24 +21,38 @@ class OpenclHeaders < Formula
 
   depends_on "cmake" => :build
 
+  resource "opencl-clhpp-headers" do
+    url "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2022.09.30.tar.gz"
+    sha256 "999dec3ebf451f0f1087e5e1b9a5af91434b4d0c496d47e912863ac85ad1e6b2"
+  end
```
Comment (SMillerDev) [inline Formula/opencl-headers.rb:27]: Isn't this just a different project?

Comment (ilya-lavrenov) [reply Formula/opencl-headers.rb:27]: OpenCL C and C++ headers are sometimes distributes in a single (CentOS 7 as an example), sometimes as two packages where C++ depends on C.

Do you think it's better to create a dedicated package for C++ headers which depends on C?

Comment (SMillerDev) [reply Formula/opencl-headers.rb:27]: Seeing how upstream treats it as two projects I think Homebrew should too.

Comment (ilya-lavrenov) [reply Formula/opencl-headers.rb]: done, thanks for the suggestion.

**Comment by cho-m** [comment]
Noting that there is another open PR for this formula - #121683.

**Comment by ilya-lavrenov** [comment]
> Noting that there is another open PR for this formula - #121683.

do you have some preferences how to resolve this conflict? :) 
My PR is ready to merge.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ilya-lavrenov, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**